core/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -242,6 +247,9 @@ private Path unzip(Path zip, Path pluginsDir) throws IOException {
     /** Load information about the plugin, and verify it can be installed with no errors. */
     private PluginInfo verify(Path pluginRoot, Environment env) throws Exception {
         // read and validate the plugin descriptor
+        if (Files.exists(pluginRoot.resolve(PluginInfo.ES_PLUGIN_PROPERTIES)) == false) {
```

Comment:
I don't think this is a usage error (it's not something the user did wrong, it is something wrong with the plugin). We also already get a FNFE when the descriptor is missing, is this really necessary? I don't think this message, vs FNFE, will give any better indication to the user that the plugin is broken.